ts/nni_manager/common/manager.ts

```diff
@@ -59,7 +59,7 @@ abstract class Manager {
     public abstract getMetricDataByRange(minSeqId: number, maxSeqId: number): Promise<MetricDataRecord[]>;
     public abstract getLatestMetricData(): Promise<MetricDataRecord[]>;
 
-    public abstract getTrialLog(trialJobId: string, logType: LogType): Promise<string>;
+    public abstract getTrialFile(trialJobId: string, fileName: string): Promise<Buffer | string>;
```

Comment:
The behavior would be more predictable to split into get text and get binary.
If you have time, not mandatory...

Reply:
I'm having a hard time thinking of another meaningful name.
Let's do it in future when we actually need this one.

Reply:
this is a broken change